servers/src/main/java/tachyon/worker/file/FileWorkerMasterSyncExecutor.java

```diff
@@ -82,9 +82,9 @@ public void heartbeat() {
     }
     Preconditions.checkState(command.commandType == CommandType.Persist);
 
-    for (CheckpointFile checkpointFile : command.checkpointFiles) {
-      mFixedExecutionService.execute(new FilePersister(mLineageDataManager, checkpointFile.fileId,
-          checkpointFile.blockIds));
+    for (PersistFile persistFile : command.getCommandOptions().getPersistOptions().persistFiles) {
```

Comment:
Slightly unrelated to this PR, but should we check to make sure `command` is not null before the check state? I think the thread will throw a null pointer after either of the errors are caught?